packages/charts/src/chart_types/goal_chart/specs/index.ts

```diff
@@ -33,7 +33,7 @@ export interface BandFillColorAccessorInput {
 export type BandFillColorAccessor = (input: BandFillColorAccessorInput) => Color;
 
 /** @alpha */
-export type GoalLabelAccessor = LabelAccessor<BandFillColorAccessorInput>;
+export type GoalLabelAccessor = LabelAccessor<BandFillColorAccessorInput> | (() => string);
```

Comment:
Why is this necessary? This type is already acceptable to the current type `LabelAccessor<BandFillColorAccessorInput>`.

```suggestion
export type GoalLabelAccessor = LabelAccessor<BandFillColorAccessorInput>;
```

Reply:
ok sweet - 9e61b47 woo